<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFtpDialog</class>
 <widget class="QDialog" name="CFtpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CFtpDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbAddress">
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>User</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Remove Directory</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leAddress"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lePort">
     <property name="text">
      <string>21</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Local Directory</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="leRemoveDir">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Periodicity (sec)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pbLocalDir">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lePass"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lePeriodicity">
     <property name="text">
      <string>3600</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="leLocalDir">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="leUser">
     <property name="text">
      <string>anonymous</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leAddress</tabstop>
  <tabstop>lePort</tabstop>
  <tabstop>leUser</tabstop>
  <tabstop>lePass</tabstop>
  <tabstop>leRemoveDir</tabstop>
  <tabstop>leLocalDir</tabstop>
  <tabstop>pbLocalDir</tabstop>
  <tabstop>lePeriodicity</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
